frontend/packages/react-app/src/App.tsx

```diff
@@ -1,25 +1,52 @@
 import React, { Component, useContext, createContext } from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
-import TopPage from "./components/pages/TopPage";
```

Comment:
The top page is not designed in Figma but it seems necessary as a landing page.

Reply:
LP was to be cut out separately, so it is not needed here.